rules/test-title-format.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+const {visitIf} = require('enhance-visitors');
+const createAvaRule = require('../create-ava-rule');
+const util = require('../util');
+
+const create = context => {
+	const ava = createAvaRule();
+	let titleRegExp;
+	try {
+		if (context.options[0] && context.options[0].regexp) {
+			titleRegExp = new RegExp(context.options[0].regexp);
+		}
+	} catch (error) {
+		console.warn(error);
```

Comment:
I would use `context.report` for this too, so it includes the file. And then return early. Just set the node to `0:0`. Unless there's a better ESLint API to use for reports like this?

Reply:
I couldn't find a better API. Other eslint-plugins don't even bother try-catch: 
 * https://github.com/yannickcr/eslint-plugin-react/blob/6e4f43ef4739191a2157659de178d1cd9a28a42e/lib/rules/boolean-prop-naming.js#L57
 * https://github.com/benmosher/eslint-plugin-import/blob/70a59fe1880b4061cd7c0f032c0b05c728223a0f/utils/moduleVisitor.js#L19

Reply:
Yeah, same with ESLint. I think it's better to just drop the try/catch then and let it crash if invalid regex.